content/docs/machine-learning/creating-a-kubeflow-cluster.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 2
+title: Creating a Kubeflow cluster
+description: Create a Civo machine learning cluster running Kubeflow. Choose from the options that suit your needs.
+---
+
+<head>
+  <title>Creating a Kubeflow cluster | Civo Documentation</title>
+</head>
+
+:::note
+Civo machine learning clusters are [billed hourly](../account/billing.md) and count towards your [account quota](../account/quota.md).
+:::
```

Comment:
Sounds good as a starting point!

Reply:
For the verification that the pods are running, this would need instructions on how the user can get the KUBECONFIG file to connect to the cluster. Do we provide KUBECONFIG files for KFaaS clusters? Can the user access the cluster directly like this using kubectl?

Reply:
Ah, yes, I do need to verify that. I haven't been able to get a Kfaas instance running to verify that 🤔

Reply:
Giving this a try in the research org on Civo, if Kfaas cluster creations works there